CMakeLists.txt

```diff
@@ -42,10 +95,8 @@ foreach(_file ${ODB_MAPPING_CLASSES})
     )
     list(APPEND MOD_ODB_SOURCES ${right_place_gencppfile})
 endforeach()
-
-add_library(mod_odb ${MOD_ODB_SOURCES})
-
-
+add_library(mod_odb STATIC ${MOD_ODB_SOURCES})
```

Comment:
Does this mean that only static builds will be supported?

Reply:
this is just to ensure the internal libraries are built statically regardless of static or dynamic build of validator. should i change it for dynamic build ?